associacao-brasileira-de-normas-tecnicas-ufrgs.csl

```diff
@@ -691,15 +687,15 @@
     <layout>
       <choose>
         <if type="article">
-          <!-- Documento (Zotero) e Programa de computador (Mendeley) -->
+          <!-- Preprint (Zotero) e Programa de computador (Mendeley) -->
           <group>
             <text macro="author" suffix=". "/>
             <text variable="authority" suffix=". "/>
             <text macro="title" suffix=" "/>
             <text variable="medium" suffix=". "/>
-            <!--incluído para ter a versão de programa de computador no mendeley-->
+            <!-- incluído para ter a versão de programa de computador no mendeley -->
             <text variable="number" prefix="Versão " suffix=". "/>
```

Comment:
That's doesn't work anymore -- number gets (properly) mapped to archive ID (e.g. arXiv ID) for preprints in Zotero. You can't use it for "Version" for `article`, sorry.

Reply:
Thanks, @adam3smith ! variable excluded.